tslib/package.json

```
@@ -7,7 +7,8 @@
     "build": "tsc -b --v",
     "build:watch": "tsc -b --v -w",
     "rebuild": "yarn clean && yarn build",
-    "lint": "standard && tslint --project .",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint:autofix": "yarn eslint . --ext .js,.jsx,.ts,.tsx --fix",
```

Save some keystrokes and get closer parity to the original flag?

```suggestion
    "lint:fix": "yarn eslint . --ext .js,.jsx,.ts,.tsx --fix",
```